wiki/Website-Markdown.md

```diff
@@ -191,6 +192,28 @@ Some text content here
 Some other content here
 ```
 
+#### Video
+
+Video assets are not natively supported in markdown, but you can easily insert a `.mp4` or similar file type into a markdown with a basic understanding of HTML:
```

Comment:
```suggestion
Video assets are not natively supported in markdown, but you can easily insert an `.mp4` or similar file type into a markdown file with a basic understanding of HTML:
```

Reply:
I think adopting this style for file names could make sense:

"[When referring to the file type, use all caps and no period. When referring to file extensions, precede with a period, then lowercase the extension acronym.](https://canvas.workday.com/content/language-and-style/file-names-types-extensions/#:~:text=In%20text%2C%20use%20title%20case,then%20lowercase%20the%20extension%20acronym.)"

So in that case the above should probably be "...easily insert an MP4 or similar file type..."

Reply:
That's a good find! I'll update to use this style, maybe something we should also add to the writing docs 🤔